install_ads

```diff
@@ -297,6 +297,10 @@ function edit_shellrc() {
     fi
     # add line
     echo ""
+    # Add ads_activate alias
+    echo "alias ads_activate=\"source $ADS_DIR/ads_conda/bin/activate $ADS_DIR/ads_conda/envs/venv_ads/venv_ads/\""
```

Comment:
Oh. That's embarrassing. `venv_ads` is there twice. I assume that's the issue? @mathieuboudreau